experiments/ClimaCore/sea_breeze/run.jl

```diff
@@ -86,13 +86,17 @@ lnd_Y_default, lnd_domain = lnd_init(xmin = 0, xmax = 500, helem = 10, npoly = 0
 # Build remapping operators
 atm_boundary = Spaces.level(atm_domain.hv_face_space, PlusHalf(0))
 
-atm_to_ocn = Operators.LinearRemap(ocn_domain, atm_boundary)
-atm_to_lnd = Operators.LinearRemap(lnd_domain, atm_boundary)
-ocn_to_atm = Operators.LinearRemap(atm_boundary, ocn_domain)
-lnd_to_atm = Operators.LinearRemap(atm_boundary, lnd_domain)
+maps = (
+    atmos_to_ocean = Operators.LinearRemap(ocn_domain, atm_boundary),
```

Comment:
minor, but we should be consistent between using "atm" and "atmos", "land" and "lnd, etc.